### DISCUSSION TITLE
No "Year" in translation file.

### DISCUSSION THREAD

**Discussion by pvanthony** [romcal/romcal#346] [Ideas]
Was trying to do translation in Tamil for the word "Year" to be used in Year_A and Year_2 but Year is not there in the translation file. 
Maybe I missed it. First time trying to do translation.

Need advice on how to create a translation for "Year"

**Comment by tukusejssirs** [comment]
Currently, we don’t localise the Sunday and ferial cycle names (see #233), but it should be fairly easy to do it in your (client) app using either `if` or `swich()`.

**Reply by emagnier** [reply]
Yes, these properties are currently not localized, but it's quick & easy to make them. I'll do it as soon as I've a bit more time for the romcal project.

**Reply by pvanthony** [reply]
> Currently, we don’t localise the Sunday and ferial cycle names (see #233), but it should be fairly easy to do it in your (client) app using either `if` or `swich()`.

Did the following and it seems to work.
```
var yearCycle = {
    YEAR_A:"ஆண்டு A",
    YEAR_B:"ஆண்டு B",
    YEAR_C:"ஆண்டு C",
    YEAR_1:"ஆண்டு 1",
    YEAR_2:"ஆண்டு 2",
}
```
To call the translation.
```
yearCycle[result[0].cycles.sundayCycle]
```
Thanks again for the advice and help. I do appreciate it.

**Reply by emagnier** [reply]
@pvanthony: FYI, the translation of these cycle metadata have been added in the last dev release (3.0.0-dev.10).


For example in English:
https://github.com/romcal/romcal/blob/0a59c5ad0385a5d00f9f36cbb61e2083912df3bd/lib/locales/en.ts#L62-L74

**Reply by tukusejssirs** [reply]
Thanks, @emagnier! And, Happy New Year to you all! :fireworks:
